code/mathematical_algorithms/src/prime_numbers_of_n/prime_numbers_of_n.js

```diff
@@ -0,0 +1,28 @@
+function primeFactors(number)
+{
+    let factors = [];
+    while (number % 2 == 0) {
+        factors.push(2)
+        number = number / 2;
+    }
+
+    for (let i = 3; i <= Math.sqrt(number); i += 2)
```

Comment:
I would move `Math.sqrt` to its own variable.

Reply:
What needs to be fixed? Can I take this?

Reply:
@tushar-nitave Since this is someone else's pull request, let that person take care of it. Thanks!

Reply:
@arnavb you mean
```javascript
let sqrt = Math.sqrt;
```

Reply:
Yup, you never know when loops may reevaluate that part.